listenbrainz/spark/request_queries.json

```diff
@@ -1,17 +1,37 @@
 {
-    "stats.user.all": {
-        "name": "stats.user.all",
-        "description": "All user statistics that ListenBrainz calculates for all ListenBrainz users",
-        "params": []
-    },
-    "stats.user.for_user": {
-        "name": "stats.user.for_user",
-        "description": "All user statistics that ListenBrainz calculates for the specified user",
-        "params": ["musicbrainz_id"]
-    },
-    "import.dump.full": {
-        "name": "import.dump.full",
-        "description": "Import a new full dump into the spark cluster",
-        "params": []
-    }
+  "stats.user.all": {
+    "name": "stats.user.all",
+    "description": "All user statistics that ListenBrainz calculates for all ListenBrainz users",
+    "params": []
+  },
+  "stats.user.artist.last_week": {
+    "name": "stats.user.artist.last_week",
```

Comment:
the `last_week` name could be misleading here, we're actually calculating the stats for the most recent week that we have in the spark cluster. I'd just rename this `week` everywhere and similar for other time frames.